extraTs/utils/permissions.ts

```diff
@@ -0,0 +1,19 @@
+function hasPermission(moduleName,role:string,permissionType:string){
```

Comment:
Type of moduleName and return type of function is missing.

Reply:
done